startup/GafferUI/gafferDispatchUICompatibility.py

```diff
@@ -1,46 +0,0 @@
-##########################################################################
-#
-#  Copyright (c) 2015, Image Engine Design Inc. All rights reserved.
-#
-#  Redistribution and use in source and binary forms, with or without
-#  modification, are permitted provided that the following conditions are
-#  met:
-#
-#      * Redistributions of source code must retain the above
-#        copyright notice, this list of conditions and the following
-#        disclaimer.
-#
-#      * Redistributions in binary form must reproduce the above
-#        copyright notice, this list of conditions and the following
-#        disclaimer in the documentation and/or other materials provided with
-#        the distribution.
-#
-#      * Neither the name of John Haddon nor the names of
-#        any other contributors to this software may be used to endorse or
-#        promote products derived from this software without specific prior
-#        written permission.
-#
-#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
-#  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
-#  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
-#  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
-#  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
-#  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
-#  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
-#  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
-#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
-#  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-#  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-#
-##########################################################################
-
-import GafferUI
-import GafferDispatchUI
-
-# Backwards compatibility - import things from GafferDispatchUI into
-# the GafferUI namespace where appropriate.
-
-for name in dir( GafferDispatchUI ) :
-	if name.endswith( "__" ) :
-		continue
-	setattr( GafferUI, name, getattr( GafferDispatchUI, name ) )
```

Comment:
https://github.com/GafferHQ/gaffer/blob/e21ae3a2c6ff98437b24056f905e6b365eaba21f/python/GafferDispatchUI/DispatcherUI.py#L133
https://github.com/GafferHQ/gaffer/blob/e21ae3a2c6ff98437b24056f905e6b365eaba21f/python/GafferDispatchUI/WedgeUI.py#L67
https://github.com/GafferHQ/gaffer/blob/e21ae3a2c6ff98437b24056f905e6b365eaba21f/python/GafferDispatchUI/WedgeUI.py#L71

Reply:
Fixed in 276b139.